addon/components/frost-fixed-table.js

```diff
@@ -421,10 +423,26 @@ export default Component.extend({
     }
   },
 
+  setShift (event) {
+    if (!this.isDestroyed) {
+      this.set('_isShiftDown', event.shiftKey)
+    }
+  },
+
   // == DOM Events ============================================================
 
   // == Lifecycle Hooks =======================================================
 
+  init () {
+    this._super(...arguments)
+    this._keyHandler = this.setShift.bind(this)
+    $(document).on(`keyup.${this.elementId} keydown.${this.elementId}`, this._keyHandler)
+  },
+
+  willDestroy () {
+    $(document).off(`keyup.${this.elementId} keydown.${this.elementId}`, this._keyHandler)
```

Comment:
Definitiely missing call to _super, but not sure what you mean by if we should release the binding on destroy. This is what we do in ember-frost-list https://github.com/ciena-frost/ember-frost-list/blob/master/addon/components/frost-list.js#L162 and it seems like the logical hook to do this https://guides.emberjs.com/v2.14.0/components/the-component-lifecycle/#toc_detaching-and-tearing-down-component-elements-with-code-willdestroyelement-code